jsoniter-scala-benchmark/src/main/scala/com/github/plokhotnyuk/jsoniter_scala/benchmark/TwitterAPIWriting.scala

```diff
@@ -32,6 +33,15 @@ class TwitterAPIWriting extends TwitterAPIBenchmark {
   @Benchmark
   def jacksonScala(): Array[Byte] = jacksonMapper.writeValueAsBytes(obj)
 
+  @Benchmark
+  def jawnCompact(): Array[Byte] = FastRenderer.render(jValue).getBytes(UTF_8)
+
+  @Benchmark
+  def jawnJsoniterScalaCompact(): Array[Byte] = writeToArray(obj)
```

Comment:
This code is the same as the `jsoniterScala` benchmark. Is that expected?

Reply:
Nice catch! Fixed in a subsequent commit...